misc/json-to-record-converter/src/main/java/io/ballerina/jsonmapper/JsonToRecordMapper.java

```diff
@@ -237,8 +237,7 @@ private static void generateRecords(JsonObject jsonObject, String recordName, bo
                                         List<DiagnosticMessage> diagnosticMessages,
                                         boolean isNullAsOptional) {
         Token recordKeyWord = AbstractNodeFactory.createToken(SyntaxKind.RECORD_KEYWORD);
-        Token bodyStartDelimiter = AbstractNodeFactory.createToken(isClosed ? SyntaxKind.OPEN_BRACE_PIPE_TOKEN :
-                SyntaxKind.OPEN_BRACE_TOKEN);
+        Token bodyStartDelimiter = AbstractNodeFactory.createToken(SyntaxKind.OPEN_BRACE_PIPE_TOKEN);
```

Comment:
Since we will not generate open records, do we need the `isClosed` param? Since this API is being used in the FE, we can deprecate it and create a new method without `isClosed`.

WDYT @NipunaRanasinghe ?